src/test-data/users.js

```diff
@@ -0,0 +1,8 @@
+const user = {
```

Comment:
What if you have more than one user? It would be better to create an object of users

Reply:
```
export const users = {
    standardUser: {
        username: 'standard_user',
        password: 'secret_sauce',
        role: 'standard',
    },
};
```